chimera.mk

```diff
@@ -13,8 +13,11 @@ update_plic: $(CHS_ROOT)/hw/rv_plic.cfg.hjson
 	sed -i 's/src: .*/src: $(PLIC_NUM_INTRS),/' $<
 	sed -i 's/target: .*/target: $(PLICCORES),/' $<
 
+gen_idma_hw:
+	make -C $(IDMA_ROOT) idma_hw_all
+
 .PHONY: chs-hw-init
-chs-hw-init: update_plic
+chs-hw-init: update_plic gen_idma_hw
```

Comment:
I believe the DMA has an automatic action that creates a separate `__deploy__*` tag for each branch that should have all the files generated. While I don't disagree that we can also just re-generate the iDMA HW here, might be good to mention that it is not always necessary.

Reply:
Thanks for letting me know! I think I will leave it like this for now, if you don't mind

Reply:
Yes, sure! Maybe a comment in the code might help for future reference, but fine with keeping it!